<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zgnep_whiteBoard</class>
 <widget class="QWidget" name="zgnep_whiteBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_Color">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src/Color.PNG</normaloff>:/src/Color.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src/save.png</normaloff>:/src/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src/delete.png</normaloff>:/src/delete.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Exit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/src/revocation-three.png</normaloff>:/src/revocation-three.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>-10</y>
     <width>398</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_StraightLine">
      <property name="toolTip">
       <string>添加直线</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/src/Figure-Line.png</normaloff>:/src/Figure-Line.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_eraser">
      <property name="toolTip">
       <string>删除</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/src/rubber.png</normaloff>:/src/rubber.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btn_round">
      <property name="toolTip">
       <string>添加圆形</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/src/Figure-circle.png</normaloff>:/src/Figure-circle.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_rectangle">
      <property name="toolTip">
       <string>添加矩形</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/src/Figure-Rectangle.png</normaloff>:/src/Figure-Rectangle.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_Pen">
      <property name="toolTip">
       <string>手写</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/src/brush.png</normaloff>:/src/brush.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>0</y>
     <width>516</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>笔尖粗细</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_size">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>45</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>99</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>笔迹</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_size_2">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>实线</string>
         </property>
         <property name="icon">
          <iconset theme="111">
           <normaloff>E:/downloads/ch01</normaloff>E:/downloads/ch01</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>虚线</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>点</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>点线</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>点点线</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>aakar</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>橡皮大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_eraeSize">
        <item>
         <property name="text">
          <string>小</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>当前模式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_model">
        <property name="text">
         <string>随心画</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_erasePic">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>qebug::</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
